<launch>
	<param name="save_dir" type="string" value="/media/why/LENOVO_USB_HDD/ir_rgb_registration_data_collect/" />
	<param name="prefix" type="string" value="0003" />
	<node name="ir_snapshot" output="screen" pkg="collect_data" required="true" type="ir_snapshot" />
	<node name="rgb_snapshot" output="screen" pkg="collect_data" required="true" type="rgb_snapshot" />
	<node args="/livox/lidar/ --duration=10 -O /media/why/LENOVO_USB_HDD/ir_rgb_registration_data_collect/0003_lidar" name="rosbag_record_lidar" output="screen" pkg="rosbag" required="true" type="record" />
</launch>
